design-system/react/src/components/PasswordStrength/PasswordStrength.tsx

```diff
@@ -0,0 +1,114 @@
+import { Check, X } from 'phosphor-react';
+import { useMemo } from 'react';
+
+import { Box } from '../Box';
+import { Flex } from '../Flex';
+import { Heading } from '../Heading';
+import { Icon } from '../Icon';
+import type { PopoverProps } from '../Popover';
+import { Popover } from '../Popover';
+import { Text } from '../Text';
+
+import { PasswordDictionary } from './constants';
+import { passwordStrengthStyles } from './styles';
```

Comment:
I think this could be just styles instead this big variable `passwordStrengthStyles` and you can export each style inside the file and use `import * as styles`